src/InnerImageZoom/InnerImageZoom.js

```diff
@@ -100,12 +100,10 @@ const InnerImageZoom = ({
   };
 
   const handleDragStart = (e) => {
-    imgProps.current.offsets = getOffsets(
-      e.pageX || e.changedTouches[0].pageX,
-      e.pageY || e.changedTouches[0].pageY,
-      zoomImg.current.offsetLeft,
-      zoomImg.current.offsetTop
-    );
+    const pageX = typeof e.pageX === 'number' ? e.pageX : e.changedTouches[0].pageX;
+    const pageY = typeof e.pageY === 'number' ? e.pageY : e.changedTouches[0].pageY;
+    //console.log(`Start pageX: ${pageX} pageY: ${pageY}`);
```

Comment:
This check is a good idea! The commented out code above could probably be removed.

Reply:
Yes, the console.log was just for verifiying the bug and is not needed.